### PR TITLE
User management

### PR DESCRIPTION
**Pull request by Rinbo**
PT: https://www.pivotaltracker.com/story/show/161472918

- Add user management page that displays users in database
- Add view for editing user roles
- Add user management controller and corresponding crud actions
- Add link to user management page on dashboard that is only visible for editors
- Add before action for user management that redirects all other users except editors that try to gain access to user management site

## Screenshots
screenshot from 2018-10-28 15-31-27

screenshot from 2018-10-28 15-31-44

### CODE REVIEW

File: app/controllers/admin/user_management_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Admin::UserManagementController < ApplicationController
```
Comment (tochman) [inline app/controllers/admin/user_management_controller.rb:1]: Convention dictates that you name this Admin::UsersController

**Review by Noelryn — APPROVED** [review]
I don't see any thing wrong

**Review by tochman — COMMENTED** [review]
Awesome job.

File: app/controllers/admin/user_management_controller.rb (hunk 1 above)
Comment (Rinbo) [reply app/controllers/admin/user_management_controller.rb:1]: Roger that

**Pull request merged**